src/main/java/com/epam/lowcost/db/DBinitialize.java

```diff
@@ -3,77 +3,72 @@
 
 import org.apache.commons.dbcp2.BasicDataSource;
 
-import java.sql.Connection;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
 
 public class DBinitialize {
 
-    private BasicDataSource basicDataSource;
     private Connection conn;
     private Statement stm;
 
     public DBinitialize(BasicDataSource basicDataSource) throws SQLException {
-        this.basicDataSource = basicDataSource;
+
         conn = basicDataSource.getConnection();
         stm = conn.createStatement();
-        createTable();
-        populateTable();
-        selectFromTable();
-
-    }
-
-
-
-    public void createTable() throws SQLException {
-
-        String sql =  "CREATE TABLE   REGISTRATION " +
-                "(id INTEGER not NULL, " +
-                " first VARCHAR(255), " +
-                " last VARCHAR(255), " +
-                " age INTEGER, " +
-                " PRIMARY KEY ( id ))";
-
-        stm.executeUpdate(sql);
-
+        initiateDB();
+        printAllDb();
     }
-    public void populateTable() throws SQLException{
-        String sql = "INSERT INTO Registration " + "VALUES (100, 'Zara', 'Ali', 18)";
-
-        stm.executeUpdate(sql);
-        sql = "INSERT INTO Registration " + "VALUES (101, 'Mahnaz', 'Fatma', 25)";
 
-        stm.executeUpdate(sql);
-        sql = "INSERT INTO Registration " + "VALUES (102, 'Zaid', 'Khan', 30)";
 
-        stm.executeUpdate(sql);
-        sql = "INSERT INTO Registration " + "VALUES(103, 'Sumit', 'Mittal', 28)";
+    private void initiateDB() throws SQLException {
 
-        stm.executeUpdate(sql);
-        System.out.println("Inserted records into the table...");
+        File fl = new File(this.getClass().getResource("/createTable").getFile());
+        try (BufferedReader br = Files.newBufferedReader(fl.toPath())) {
 
+            String sql = br.readLine();
+            while (sql != null) {
+                stm.addBatch(sql);
+                sql = br.readLine();
+            }
```

Comment:
https://www.mkyong.com/java8/java-8-stream-read-a-file-line-by-line/
You can try to use streams

Reply:
Changed initiated method to stream realization.